Added in providerVisualization and supporting styles.

schreiaj opened the pull request:
Also updated the provider api (and documentation) to support displaying the parent name without an additional network call.

Commit:
Added in providerVisualization and supporting styles.

Also updated the provider api (and documentation) to support displaying the parent name without an additional network call.

carpeliam commented on app/assets/javascripts/provider_chart.js.coffee:
```diff
@@ -0,0 +1,188 @@
+window.PopHealth ||= {}
+PopHealth.vis ||= {}
```
This should be `PopHealth.viz`, right?

carpeliam commented on app/assets/javascripts/application.js.coffee:
```diff
@@ -14,6 +14,7 @@
 #= require config
 #= require helpers
 #= require population_chart
+#= require provider_chart
```
I wonder if, now that we have multiple visualizations, it makes sense to create a `viz` directory, and do `#= require_tree ./viz`. I'm ok either way, though.

carpeliam commented on app/assets/stylesheets/vis.less:
```diff
@@ -0,0 +1,46 @@
+svg{
```
This should be namespaced.

carpeliam commented on app/assets/javascripts/templates/providers/show.hbs:
```diff
@@ -1,18 +1,25 @@
+	  <dd>{{team.name}}</dd>
+	</dl>
+</div>
+<div id="providerChart"></div>
```
This should have a class of `col-md-9`, at least for consistency, right?

Commit:
Namespaced sag styling to the providerChart. 

Fixed minor typo in the  visualization wherein we were initializing an extra object

Added in class on the chart for consistency.
